interview-questions.md

```diff
@@ -8,44 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: In a Ruby class, super would be used to create a sub class.
 
-Researched answer:
+Researched answer: Super is a keyword that gets called within a parent class and searches for another method with the same name as the parent
```

Comment:
[This](https://github.com/learn-academy-2023-echo/syllabus/blob/main/ruby/inheritance.md) may give you a better understanding of what super is used for